src/app/(mobile-ui)/withdraw/manteca/page.tsx

```
@@ -0,0 +1,440 @@
+    }, [selectedMethodType])
+
+    const validateDestinationAddress = async (value: string) => {
+        value = value.trim()
+        if (!value) {
+            return false
+        }
+
+        let isValid = false
+        switch (countryPath) {
+            case 'argentina':
+                isValid = validateCbuCvuAlias(value)
```

Biome (2.1.2)

[error] This case is falling through to the next case.

Add a `break` or `return` statement to the end of this case to prevent fallthrough.

(lint/suspicious/noFallthroughSwitchClause)